### PR TITLE
Add German translations

### PR DESCRIPTION
**Pull request by johanndev**

### CODE REVIEW

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.de.xlf [hunk 1]
```diff
@@ -4,387 +4,387 @@
     <body>
       <trans-unit id="ID3000">
         <source>The security token is missing.</source>
-        <target state="new">The security token is missing.</target>
+        <target state="new">Der Security-Token fehlt.</target>
```
Comment (kevinchalet) [inline src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.de.xlf]: Can you please replace all the `state` attributes of the translated strings from `new` to `translated`?

Thanks.

**Review by kevinchalet — APPROVED** [review]
Thanks a lot for your contribution, @johanndev!

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.de.xlf (hunk 1 above)
Comment (johanndev) [reply src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.de.xlf]: Done!

**Comment by kevinchalet** [comment]
Merged, thanks for your PR and congrats for your first contribution here! 👏

**Comment by johanndev** [comment]
Thank you! :)

And thanks for the great set of libraries!

**Pull request merged**